Objects/typeobject.c

```diff
@@ -4094,19 +4092,29 @@ _PyStaticType_Dealloc(PyTypeObject *type)
     Py_CLEAR(type->tp_cache);
     // type->tp_subclasses is NULL
 
+    // PyObject_ClearWeakRefs() raises an exceptions if Py_REFCNT() != 0
```

Comment:
Grammar error/typo in the comment.

Reply:
"an exceptions", oh I see. I will fix it later. First, I need to merge https://github.com/python/cpython/pull/30809 and then rebase this PR on top of it.